tests/test_crypto.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Unit tests for crypto.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# vendor libraries
+import pytest
+from os import linesep
+from pathlib import Path
+
+# local libraries
+from appcli.crypto import crypto
+from appcli.crypto.cipher import Cipher
+
+
+def test_decrypt_values_in_file(tmpdir):
+    key_file = Path(tmpdir, "key")
+    crypto.create_and_save_key(key_file)
```

Comment:
I'm not a fan of the fact that our test relies upon a newly-generated cipher key every time it's run